includes/admin/tools.php

```diff
@@ -1614,6 +1614,35 @@ function edd_tools_sysinfo_download() {
 }
 add_action( 'edd_download_sysinfo', 'edd_tools_sysinfo_download' );
 
+/**
+ * Redirects requests to the old sales log to the orders page.
+ *
+ * @since 3.0
+ */
+function edd_redirect_sales_log() {
+	if ( edd_is_admin_page( 'tools', 'logs' ) && ! empty( $_GET['view'] ) && 'sales' === $_GET['view'] ) {
+		$query_args = array(
+			'page' => 'edd-payment-history'
+		);
+
+		$args_to_remap = array(
+			'download'   => 'product-id',
+			'start-date' => 'start-date',
+			'end-date'   => 'end-date'
+		);
+
+		foreach( $args_to_remap as $old_arg => $new_arg ) {
+			if ( ! empty( $_GET[ $old_arg ] ) ) {
+				$query_args[ $new_arg ] = urlencode( $_GET[ $old_arg ] );
+			}
+		}
+
+		wp_safe_redirect( esc_url_raw( add_query_arg( $query_args ), edd_get_admin_base_url() ) );
```

Comment:
This redirect is working but the URL ends up having both the old and new parameters in it:

```
http://robin.local/wp-admin/edit.php?post_type=download&page=edd-payment-history&tab=logs&view=sales&download=12357&product-id=12357
```
Wondering if that's just WP being wonky (feel like I was wrestling with this in the Slack extension as well) or if it's fixable?

Reply:
No -- it's me. I messed up my commas.